File: ios/RCTMGL-v10/RCTMGLTerrainManager.m
#import <React/RCTBridgeModule.h>
#import <React/RCTViewManager.h>

@interface RCTMGLTerrain : UIView
{}
@property (nonatomic) NSObject* exaggeration;
@end

@interface RCT_EXTERN_MODULE(RCTMGLTerrainManager, RCTViewManager)

RCT_EXPORT_VIEW_PROPERTY(sourceID, NSString);

RCT_EXPORT_VIEW_PROPERTY(reactStyle, NSDictionary);

@end

File: ios/RCTMGL-v10/RCTMGLTerrainManager.swift
@objc(RCTMGLTerrainManager)
class RCTMGLTerrainManager: RCTViewManager {
    @objc
    override static func requiresMainQueueSetup() -> Bool {
        return true
    }
  
    override func view() -> UIView! {
      let terrain = RCTMGLTerrain()
      terrain.bridge = self.bridge
      return terrain
    }
}
